src/VideoStreaming/VideoReceiver.cc

```diff
@@ -205,7 +205,7 @@ void VideoReceiver::start()
             }
             g_object_set(G_OBJECT(dataSource), "uri", qPrintable(_uri), "caps", caps, NULL);
         } else {
-            g_object_set(G_OBJECT(dataSource), "location", qPrintable(_uri), "latency", 0, "udp-reconnect", 1, "timeout", 5000000, NULL);
+            g_object_set(G_OBJECT(dataSource), "location", qPrintable(_uri), "latency", 17, "udp-reconnect", 1, "timeout", static_cast<guint64>(5000000), NULL);
```

Comment:
Is 17ms enough? That's half a frame (assuming 30fps). Would 35ms be any more noticeable?

Good grief on the 32 -> 64 bit. I would assume the compiler would have better sense. Good hunt result!